### PR TITLE
add DLL import/export macro on Windows

### PR DESCRIPTION
**Pull request by kejxu**
add DLL import/export macro `BONDCPP_DECL` for Bond class

### CODE REVIEW

**Comment by seanyen** [comment]
@mjcarroll Hope I am pinging the correct maintainer. Can you take a look of this and #46? We believe that's the required fixes to unblock Windows build break. (and it is verified in https://aka.ms/ros)

**Review by mjcarroll — APPROVED** [review]

**Pull request merged**
